roles/systemd-networkd/handlers/main.yml

```diff
@@ -1,7 +1,7 @@
 ---
 # handlers for ansible-role-systemd-networkd
 
-- name: restart systemd-networkd
+- name: Restart systemd-networkd
```

Comment:
If you rename a handler, you will also need to rename it in the places where it's notified

Reply:
ah, thanks for noticing, seems I forgot doing that in this one